examples/RL/ppo.py

```diff
@@ -8,10 +8,12 @@
 from stable_baselines3.common.vec_env import SubprocVecEnv
 
 from l5kit.environment.feature_extractor import CustomFeatureExtractor
+from l5kit.environment.callbacks import L5KitEvalCallback
 
 # Dataset is assumed to be on the folder specified
 # in the L5KIT_DATA_FOLDER environment variable
 # Please set the L5KIT_DATA_FOLDER environment variable
+os.environ["L5KIT_DATA_FOLDER"] = os.environ["HOME"] + '/level5_data/'
```

Comment:
Remember to remove this as it is assuming that users are putting their data at `~/level5_data`